### ISSUE TITLE
Pie Chart console error

### ISSUE THREAD

**Issue by alexiastandish** [jask-oss/reaviz#128]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior

The Pie Chart in the Reaviz library throws this error in the console

![reaviz-console-errors](https://user-images.githubusercontent.com/40964861/78084110-3ae57580-736c-11ea-928e-f1883342f17e.png)

## Expected behavior

The console error be fixed and removed - not entirely sure what the error is pertaining because I don't have much experience working with d3 or svgs but based on a little research it seems to have something to do with the line generator

## What is the motivation / use case for changing the behavior?

To prevent all the console errors from showing up for anyone using the pie chart in their app

Others:

Thank you !

**Comment by amcdnl** [comment]
I was able to reproduce this by opening the default chart demos:

```
Error: <path> attribute d: Expected moveto path command ('M' or 'm'), "[object Object]".
```

I'm thinking its probably related to the animtion's transition from 0 to N. I will look into it, probably just needs aa 0001 number or something.

**Comment by alexiastandish** [comment]
@amcdnl Thank you!

**Comment by amcdnl** [comment]
dev versions are published on every commit to master but i just released the latest 8.1.0 this morning with this fix and others.
